src/main/java/de/tum/in/www1/artemis/service/AssessmentService.java

```diff
@@ -78,7 +78,9 @@ public Result updateAssessmentAfterComplaint(Result originalResult, Exercise exe
 
         // Update the result that was complained about with the new feedback
         originalResult.updateAllFeedbackItems(assessmentUpdate.getFeedbacks());
-        originalResult.evaluateFeedback(exercise.getMaxScore());
+        if (!(exercise instanceof ProgrammingExercise)) {
```

Comment:
WIll you also refactor this service in the code duplication PR? Because I would suggest to have some inheritance here in order to avoid `instanceof` checks.

Reply:
Yes, this will be fixed in the code duplication PR